services/basic_types.proto

```diff
@@ -113,6 +113,21 @@ message AccountID {
     }
 }
 
+/**
+ * Represents an NFT on the Ledger
+ */
+message NftID {
+    /**
+     * The (non-fungible) token of which this NFT is an instance
+     */
+    TokenID tokenID = 1;
```

Comment:
should be camel case token_id

Reply:
Almost all of the names in basic_types.proto are in camel case, but a few are snake case. Do we want to standardize on snake case? If so, there are a lot more places that should be changed.

Reply:
yeah, I think old ones are in camel case. But then we standardized to snake case for any further changes. It should not affect generated code afaik.

Reply:
I fixed it here. I also created issue #7423 to fix it other places.